client/src/pages/patients/register/PatientRegistration.jsx

```diff
@@ -178,7 +178,7 @@ export default function PatientRegistration () {
         lastName,
         gender,
         language,
-        dateOfBirth: new Date(dateOfBirth),
+        dateOfBirth: dateOfBirth,
```

Comment:
With object literal shorthand, this can just be: `dateOfBirth,`

Reply:
Good point, fixed!